Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -579,15 +584,31 @@ private class TopSitesDataSource: NSObject, UICollectionViewDataSource {
         }
     }
 
+    private func setHistorySites(historySites: [Site]) {
+        self.historySites = historySites
+    }
+
+    private func filterSuggestedSites() {
```

Comment:
I think this needs to happen on `init`. Since we are calling this method after we have grabbed the history tiles from the DB here [1], when you remove the suggested tiles, switch panels, and come back, the suggested tiles flicker back then away.

[1] https://github.com/mozilla/firefox-ios/pull/1587/files#diff-abfc8d88f8fbb974154bce59e22c81a6R124